index.js

```diff
@@ -64,29 +76,40 @@ async function connect() {
       }
 
       if (currVersion !== snapshotValue.version) {
+        snapshotValue.initialLoad = initialLoad;
+
         (snapshotValue.actions || []).forEach(action => actionFunctions[action](snapshotValue));
 
         localStorage.setItem('jellySyncVersion', snapshotValue.version);
+        localStorage.setItem('jellysyncEndpoint', JSON.stringify(endpoint));
         currVersion = snapshotValue.version;
       }
+
+      initialLoad = false;
     });
   } catch (e) {
-    attempts--;
-    endpoint = await getEndpoint(pId);
-    dbRef = database.ref(`project/${endpoint}`);
-    connect();
-
-    dbRef.onDisconnect(() => connect());
+    await reconnect();
   }
 }
 
+async function reconnect() {
+  attempts--;
+  endpoint = await getEndpoint(pId);
+  dbRef = database.ref(`/projects/${pId}/${endpoint.id}`);
+  connect();
+
+  dbRef.onDisconnect(() => connect());
+}
+
 async function getEndpoint(projectId) {
-  const endpointId = await axios.get(
-    `https://us-central1-jellysync.cloudfunctions.net/api/projectEndpoint?projectId=${projectId}`
-  );
-  localStorage.setItem('jellysyncEndpoint', endpointId.data);
+  const prdUrl = 'https://us-central1-jellysync.cloudfunctions.net/api';
+  const localUrl = 'http://localhost:5001/jellysync/us-central1/api';
```

Comment:
what are we doing about these? Do we need a set-env?

Reply:
o yeah, forget about this